test/pom.xml

```diff
@@ -239,6 +263,23 @@ THE SOFTWARE.
           <skip>true</skip>
         </configuration>
       </plugin>
+      <plugin> <!-- TODO pending JENKINS-45271 fix, would be best to finish moving MavenModuleSet-specific tests to maven-plugin and delete the test dep here -->
```

Comment:
There are a bunch. I do think it would be wise to move them to `maven-plugin` and delete the dep. I just did not want to take it on in this PR; would confuse the issue too much.